### PR TITLE
Remove outdated whatsnew pages [no bug]

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Removes templates, assets, and tests for WNPs 79-85, and also removes the experiment bits from WNP88 (none of the variants outperformed the control).

## Testing
Make sure I didn't miss anything or delete anything still in use somewhere.

### CODE REVIEW

File: bedrock/base/templates/includes/protocol/navigation/menus/whoweare.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h4 class="c-menu-item-title">{{ ftl('navigation-v2-careers', fallback='navigati
           </li>
           <li>
             <section class="c-menu-item mzp-has-icon">
-              <a class="c-menu-item-link" href="http://blog.mozilla.org/?{{ utm_params }}" data-link-name="Mozilla Blog" data-link-type="nav" data-link-position="topnav" data-link-group="who-we-are">
+              <a class="c-menu-item-link" href="https://blog.mozilla.org/?{{ utm_params }}" data-link-name="Mozilla Blog" data-link-type="nav" data-link-position="topnav" data-link-group="who-we-are">
```
Comment (craigcook) [inline bedrock/base/templates/includes/protocol/navigation/menus/whoweare.html:65]: Just gonna sneak this in...

**Review by alexgibson — CHANGES_REQUESTED** [review]
Changes look good, nice clean up 💯 

I noticed a couple of stray assets:

- Some images in the media directory? https://github.com/mozilla/bedrock/tree/master/media/img/firefox/whatsnew
- The tests that are being removed can also have their associated page objects cleaned up: https://github.com/mozilla/bedrock/tree/master/tests/pages/firefox/whatsnew

**Comment by craigcook** [comment]
See, I knew I'd forget something... updated!

**Comment by craigcook** [comment]
There was a test failure for WNP86 in French. Originally we were serving FR the whatsnew81 template, but since that's being deleted FR was getting the standard account page, which has no S2D widget so the tests failed. We could update the view to show the standard mobile page to FR (instead of account), but I opted to just remove the tests and not do anything special for French. We could also just delete the 86 page entirely, but I figured we should wait until Fx90.

**Review by alexgibson — APPROVED** [review]

**Comment by craigcook** [comment]
Successful test run: https://gitlab.com/mozmeao/www-config/-/pipelines/319456041

**Pull request merged**